tools/scxtop/src/lib.rs

```diff
@@ -204,3 +204,40 @@ pub enum Action {
     Up,
     None,
 }
+
+impl std::fmt::Display for Action {
```

Comment:
nit: should we add `strum_macros::Display` and derive this? It'll cut down on boilerplate, but your call.